core/src/main/java/org/geysermc/geyser/GeyserImpl.java

```diff
@@ -414,9 +414,6 @@ private void startInstance() {
             }
         }
 
-        // Ensure that PacketLib does not create an event loop for handling packets; we'll do that ourselves
-        TcpSession.USE_EVENT_LOOP_FOR_PACKETS = false;
```

Comment:
Q: This is redundant as the event loop is now created by Geyser and passed directly to mcpl, thereby not creating one there, correct?

Reply:
yes, exactly that